src/index.ts

```diff
@@ -32,6 +32,12 @@ const options = {
         default: false,
         description: 'Should imports be separated by new line?',
     },
+    importOrderGroupNamespaceSpecifiers: {
+        type: 'boolean',
+        category: 'Global',
+        default: false,
+        description: 'Should namespace specifies be grouped at the top?',
```

Comment:
I would like to have this feature. What do you think about a different approach here? May be we can do it by exposing a special keyword similar to [`THIRD_PARTY_MODULES` ](https://github.com/trivago/prettier-plugin-sort-imports#importorder). There can be cases where some people would like to keep them at a different locations in the import order.

Reply:
I agree, I would also occasionally like to group all namespace specifiers at the top in their own group, but I also like this current solution which would order the namespace specifiers at the top of each group. Is there a solution to suppport both?

Reply:
>I would also occasionally like to group all namespace specifiers at the top in their own group, but I also like this current solution which would order the namespace specifiers at the top of each group. 


I think I misunderstood your PR here. That is fine.  Could you please add this in Readme. md.

Reply:
Yes sorry, I think I had mislabelled my description. I've Added it to the README.

So this PR adds the ability to sort namespace specifiers to the top of the import group.